gameList.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+$db = new PDO ('mysql:host=db;dbname=collection', 'root', 'password');
+
+$db -> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
Suggesting putting this into a nice little function that can be used to grab a DB connection whenever you need it